api/routes/skills-route.ts

```diff
@@ -6,24 +6,43 @@ const router = express.Router()
 import skillsWidget from '../../src/widgets/skills'
 import { getBoolean } from '../../src/utils'
 import errorWidget from '../../src/widgets/error'
-
-// Primary skills route
+/**
+ * Primary skill path.
+ * 
+ * This path is for the primary skill screen. It has the following possible queries
+ * @param languages the list of languages to show.
+ * @param frameworks the list of frameworks to show.
+ * @param libraries the list of libraries to show.
+ * @param names @deprecated the full list of all skills it needs to show. deprecated from version 1.1.0 onwards.
+ */
 router.get('/', function (req: Request, res: Response) {
-    const { names, includeNames } = req.query
+    const { names, languages, frameworks, libraries, tools, software, includeNames} = req.query
 
     // Set the header's type to svg/xml
     res.setHeader('Content-Type', 'image/svg+xml')
 
     // Check if languages argument is not present
-    if (names === undefined || names === null) {
+    if ((languages === undefined || languages === null) 
```

Comment:
Instead of e.g. `languages === undefined || languages === null`, you can use `!languages`. Does the exact same thing.